frontend/src/components/VTag/VTag.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <VButton
+    as="VLink"
+    size="small"
+    variant="filled-gray"
+    class="label-bold"
+    :href="href"
+    >{{ title }}</VButton
+  >
+</template>
+
+<script lang="ts">
+import { defineComponent } from "@nuxtjs/composition-api"
```

Comment:
```suggestion
import { defineComponent } from "vue"
```

We are trying to move away from `@nuxtjs/composition-api` wherever possible: this will make the transition to Nuxt 3 easier. This is why we only use the `defineComponent` imported from `@nuxtjs/composition-api` on `page`s. On a component like this, it's better to import it from `vue`.